samples/javascript_nodejs/01.console-echo/index.js

```diff
@@ -15,13 +12,17 @@ require('dotenv').config({ path: ENV_FILE });
 // We are using the ConsoleAdapter, which enables a bot you can chat with from within your terminal window.
 const adapter = new ConsoleAdapter();
 
+// Import our bot class.
+const EchoBot = require('./bot');
```

Comment:
change to:
const { EchoBot } = require('./bot');

Reply:
Love named imports.